src/Utils/MomentoToPhpRedisExceptionMapper.php

```diff
@@ -2,13 +2,32 @@
 
 namespace Momento\Cache\Utils;
 
-use Exception;
+use Momento\Cache\Errors\InvalidArgumentError;
 use Momento\Cache\Errors\NotImplementedException;
+use Momento\Cache\Errors\TimeoutError;
+use RedisException;
 
 class MomentoToPhpRedisExceptionMapper
 {
-    public static function createCommandNotImplementedException(string $command): Exception
+    /**
+     * @throws RedisException
+     */
+    public static function mapException($error): bool|RedisException
+    {
+        $sdkError = $error->asError()->innerException();
+        return match (get_class($sdkError)) {
+            TimeoutError::class => throw new RedisException("Timeout occurred: " . $sdkError->getMessage()),
+            default => false,
+        };
+    }
```

Comment:
Should rename for clarity to indicate this throws an exception in the event of a timeout; otherwise returns false.